azure/v12.0.2/README.md

```diff
@@ -0,0 +1,11 @@
+## :zap:  Giant Swarm Release 12.0.2 for Azure :zap:
+
+**If you are upgrading from 12.0.1, upgrading to this release will not roll your nodes. It will only update the apps.**
+
+This release upgrades external-dns app to v1.4.0.
```

Comment:
```suggestion
This release upgrades external-dns app to v1.4.0 to add monitoring and improve reliability.
```

Reply:
@cokiengchiara it doesn't really improve reliability. Since there are outside of the release, CP side, changes (addition of alerts, ops-recipes) would  you be OK with
```suggestion
This release upgrades external-dns app to v1.4.0 to improve observability.
```